R-notebooks/dashboard_functions.R

```diff
@@ -0,0 +1,9 @@
+plot_28_day_frequency <- function(df_to_plot, geo_type) {
+  # These are all required for rendering using covidcast.plot
+  df_to_plot$time_value = "2020-04-15"
```

Comment:
Right now, all plots are labeled with 2020-04-15. You can use the data from the covidcast_signal objects to set an appropriate value.

Reply:
Done (though honestly I think the issue date is meaningless after grouping -- would be nice to consider support plotting without an issue date set)

Reply:
Ah, my intent was mainly to update the date in the plot titles. Could you add it in addition to "last 28 days", so that it's easy to see how old the graphs are?